GitHubAction/UploadArtifactApi/HttpArtifactUploadApi.cs

```diff
@@ -39,9 +36,15 @@ public HttpArtifactUploadApi(HttpClient httpClient)
         formData.Add(new StringContent(catalog.IsPreRelease ? "true" : "false"), "isprerelease");
         formData.Add(new StringContent(catalog.CommitterMail), "developer");
         formData.Add(new StringContent(catalog.ReleaseUri), "releasepath");
-        formData.Add(new StreamContent(fileStream), "file", Path.GetFileName(fileStream.Name));
 
-        string logInfo = $"--name {name} --version {catalog.Version} --contentType {catalog.ContentType} --branch {catalog.Branch} --identifier {catalog.Identifier} --isprerelease {catalog.IsPreRelease} --developer {catalog.CommitterMail} --releasepath {catalog.ReleaseUri} --file {Path.GetFileName(fileStream.Name)}";
+        MemoryStream ms = new MemoryStream();
```

Comment:
implements IDisposable

Reply:
Not necessary as the StreamContent class is also IDisposable as is the MultipartFormDataContent. When that is being disposed, it will dispose all the content as well.